### PR TITLE
Deleting  note from ticket

### PR DESCRIPTION
**Pull request by hardy1334**
With this, we will be able to delete notes on server side.

### CODE REVIEW

**Comment by hardy1334** [comment]
@ragesoss  can you please review it for the issue https://github.com/WikiEducationFoundation/WikiEduDashboard/issues/3502 . I'll be creating separate pr in the wikiedudashboard for the client side work

**Comment by ragesoss** [comment]
@hardy1334 please include tests for this as well. There are some examples you can follow in the tickets_controller_spec.

**Comment by hardy1334** [comment]
sure, will do it

**Comment by ragesoss** [comment]
I've tested this in conjuction with https://github.com/WikiEducationFoundation/WikiEduDashboard/pull/3870 and it's working smoothly. Nice work!

I'd still like to have a spec included in the this PR. Once that is ready, I can merge this, and then you can update the other PR to include the newer version of TicketDispenser in the Gemfile.lock and then it ought to be clear to merge once code review is done.

**Comment by hardy1334** [comment]
sure I'll add the test for it, and push that commit

**Comment by hardy1334** [comment]
@ragesoss I have added spec for it can you please check it and merge it so that I make necessary changes in other pr

**Comment by hardy1334** [comment]
@ragesoss  In the gemfile.lock for the other pr, current version is **0.1.0** , so should I change it to 0.2.0 or 0.1.1 ?

File: spec/controllers/ticket_dispenser/tickets/replies_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe TicketDispenser::Tickets::RepliesController, type: :request do
+      describe '#destroy' do
```
Comment (ragesoss) [inline spec/controllers/ticket_dispenser/tickets/replies_controller_spec.rb]: This is not standard Ruby indentation. Should be two spaces.

Comment (hardy1334) [reply spec/controllers/ticket_dispenser/tickets/replies_controller_spec.rb]: I'll make the necessary changes

**Comment by ragesoss** [comment]
The version number is fine to leave as it is. The gemfile is locking it to the specific revision straight from git. You should update the bundle to this new revision as part of your PR in the other repo, now that it's merged here.

**Pull request merged**